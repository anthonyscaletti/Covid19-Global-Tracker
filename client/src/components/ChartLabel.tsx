import React from 'react';

const ChartLabel = (props: IOwnProps) => {
    return (
        <h5>{props.label}</h5>
    );
}

interface IOwnProps {
    label: string
}

export default ChartLabel;