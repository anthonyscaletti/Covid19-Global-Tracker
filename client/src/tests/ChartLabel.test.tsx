import React from 'react';
import renderer from 'react-test-renderer';

import ChartLabel from '../components/ChartLabel';

it('renders correctly', () => {
  const tree = renderer
    .create(<ChartLabel />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});