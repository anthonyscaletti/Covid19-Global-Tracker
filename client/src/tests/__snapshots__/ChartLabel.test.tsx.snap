// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="chartLabel"
>
  <div
    className="labelBox"
    style={
      Object {
        "backgroundColor": "undefined",
      }
    }
  />
  <h5>
      
  </h5>
</div>
`;
